test_runner/performance/test_branch_creation.py

```diff
@@ -108,3 +112,37 @@ def test_branch_creation_many(neon_compare: NeonCompare, n_branches: int):
         branch_creation_durations.append(dur)
 
     _record_branch_creation_durations(neon_compare, branch_creation_durations)
+
+
+# Test measures the branch creation time of branching from a timeline with a lot of relations
+# when the root timeline is busy under a workload and not busy.
```

Comment:
> when the root timeline is busy under a workload and not busy

Didn't understand that. Rephrase?

Reply:
Ah, looking at the code, now I get it. It runs the same test twice. The first time, there's a concurrent INSERT happening on the root timeline, i.e. it's busy. The second time, there's no concurrent load, i.e. it's not busy. It's not clear from the comment.